mqtt-broker.cabal

```diff
@@ -27,11 +28,14 @@ test-suite spec
   type:                exitcode-stdio-1.0
   hs-source-dirs:      test
   main-is:             Spec.hs
-  build-depends:       base
-                     , mqtt-broker
+  build-depends:       attoparsec
+                     , base
```

Comment:
Make base first as there is no chance it will be removed in the future (the whole point of the `,` at start of line is to make smaller git diffs).